.storybook-sdk/preview.tsx

```diff
@@ -16,7 +17,15 @@ export const parameters = {
   },
 };
 
-export const decorators = []; // No decorators for Embedding SDK stories, as we want to simulate real use cases
+const decorators = [
+  Story => {
+    if (!document.body.dir) {
+      document.body.dir = "ltr";
```

Comment:
isn't this un-realistic? we can't do this on customers app

I don't think we should have any custom logic in the sdk storybook, everything needed should be done by the MetabaseProvider/sdk code

Reply:
See for example: https://github.com/metabase/metabase/pull/54081 where I'm removing some imports that shouldn't be there

Reply:
Initially i thought that it is the Storybook-only issue. But seems it affects the whole SDK as well.